pkg/controller/performanceprofile/components/machineconfig/machineconfig.go

```diff
@@ -11,7 +11,7 @@ import (
 	"github.com/coreos/go-systemd/unit"
 	igntypes "github.com/coreos/ignition/config/v2_2/types"
 
-	performancev1alpha1 "github.com/openshift-kni/performance-addon-operators/pkg/apis/performance/v1alpha1"
+	performancev1alpha1 "github.com/openshift-kni/performance-addon-operators/pkg/apis/performance/v1"
```

Comment:
why do we need this rename?

Reply:
It was a temporary decision to remove v1alpha1 all-together, I don't think is needed anymore

Reply:
Wait, is not a rename, we point to another API, I will change also performancev1alpha1 -> performancev1 in all the controller code